src/App.jsx

```
@@ -4,6 +4,7 @@ import HomePage from './pages/HomePage';
 import AboutPage from './pages/AboutUs';
 import Contact from './pages/Contact';
 import Faq from './pages/Faq';
+import signup from './pages/signup.jsx'
```

ESLint

[error] 'signup' is defined but never used.

(no-unused-vars)